internalshared/listenerutil/listener.go

```diff
@@ -67,92 +72,70 @@ func UnixSocketListener(path string, unixSocketsConfig *UnixSocketsConfig) (net.
 	}, nil
 }
 
-func WrapTLS(
+func GetTLSConfig(
 	ln net.Listener,
```

Comment:
Is it still necessary to pass the net.Listener here? Seems like it’s not being used anymore

Reply:
Fixed.